src/node/optimizer/index.ts

```diff
@@ -286,6 +286,11 @@ function resolveQualifiedDeps(
       debug(`skipping ${id} (internal excluded)`)
       return false
     }
+    // #804
+    if (id.startsWith('@types/')) {
```

Comment:
Will this too special? Should we do a whitelist for it? Maybe internal or also exposing to the config.